<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>config</class>
 <widget class="QWidget" name="config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1485</width>
    <height>920</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>config</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>365</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>参数显示</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pushButton_readData">
            <property name="maximumSize">
             <size>
              <width>175</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>参数读入</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_editSuc">
            <property name="maximumSize">
             <size>
              <width>175</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>参数写出</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QTableWidget" name="tableWidget">
          <property name="maximumSize">
           <size>
            <width>350</width>
            <height>16777215</height>
           </size>
          </property>
          <row>
           <property name="text">
            <string>0</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>1</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>2</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>3</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>4</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>5</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>6</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>7</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>8</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>9</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>10</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>11</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>12</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>13</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>14</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>15</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>16</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>17</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>18</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>19</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>20</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>21</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>22</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>23</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>24</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>25</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>26</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>27</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>28</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>29</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>30</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>31</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>32</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>33</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>34</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>35</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>36</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>37</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>38</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>39</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>40</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>41</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>42</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>43</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>44</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>45</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>46</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>47</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>48</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>49</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>50</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>51</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>52</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>53</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>54</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>55</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>56</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>类型</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>名称</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>数据</string>
           </property>
          </column>
          <item row="0" column="0">
           <property name="text">
            <string>相机内参</string>
           </property>
          </item>
          <item row="0" column="1">
           <property name="text">
            <string>Fdx</string>
           </property>
          </item>
          <item row="0" column="2">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="1" column="1">
           <property name="text">
            <string>u0</string>
           </property>
          </item>
          <item row="1" column="2">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="2" column="1">
           <property name="text">
            <string>Fdy</string>
           </property>
          </item>
          <item row="2" column="2">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="3" column="1">
           <property name="text">
            <string>v0</string>
           </property>
          </item>
          <item row="3" column="2">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="4" column="0">
           <property name="text">
            <string>畸变系数</string>
           </property>
          </item>
          <item row="4" column="1">
           <property name="text">
            <string>k1</string>
           </property>
          </item>
          <item row="4" column="2">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="5" column="1">
           <property name="text">
            <string>k2</string>
           </property>
          </item>
          <item row="5" column="2">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="6" column="1">
           <property name="text">
            <string>p1</string>
           </property>
          </item>
          <item row="6" column="2">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="7" column="1">
           <property name="text">
            <string>p2</string>
           </property>
          </item>
          <item row="7" column="2">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="8" column="0">
           <property name="text">
            <string>光平面方程</string>
           </property>
          </item>
          <item row="8" column="1">
           <property name="text">
            <string>a</string>
           </property>
          </item>
          <item row="8" column="2">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="9" column="1">
           <property name="text">
            <string>b</string>
           </property>
          </item>
          <item row="9" column="2">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="10" column="1">
           <property name="text">
            <string>c</string>
           </property>
          </item>
          <item row="10" column="2">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="11" column="0">
           <property name="text">
            <string>方向向量</string>
           </property>
          </item>
          <item row="11" column="1">
           <property name="text">
            <string>dir_x</string>
           </property>
          </item>
          <item row="11" column="2">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="12" column="1">
           <property name="text">
            <string>dir_y</string>
           </property>
          </item>
          <item row="12" column="2">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="13" column="1">
           <property name="text">
            <string>dir_z</string>
           </property>
          </item>
          <item row="13" column="2">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="14" column="0">
           <property name="text">
            <string>输送链速度</string>
           </property>
          </item>
          <item row="14" column="1">
           <property name="text">
            <string>chainSP[mm/s]</string>
           </property>
          </item>
          <item row="15" column="0">
           <property name="text">
            <string>用户坐标系</string>
           </property>
          </item>
          <item row="15" column="1">
           <property name="text">
            <string>u_rx</string>
           </property>
          </item>
          <item row="16" column="1">
           <property name="text">
            <string>u_ry</string>
           </property>
          </item>
          <item row="17" column="1">
           <property name="text">
            <string>u_rz</string>
           </property>
          </item>
          <item row="18" column="1">
           <property name="text">
            <string>u_x</string>
           </property>
          </item>
          <item row="19" column="1">
           <property name="text">
            <string>u_y</string>
           </property>
          </item>
          <item row="20" column="1">
           <property name="text">
            <string>u_z</string>
           </property>
          </item>
          <item row="21" column="0">
           <property name="text">
            <string>标定处与喷涂处距离</string>
           </property>
          </item>
          <item row="21" column="1">
           <property name="text">
            <string>dis</string>
           </property>
          </item>
          <item row="22" column="0">
           <property name="text">
            <string>手眼标定</string>
           </property>
          </item>
          <item row="22" column="1">
           <property name="text">
            <string>eh_nx</string>
           </property>
          </item>
          <item row="23" column="1">
           <property name="text">
            <string>eh_ny</string>
           </property>
          </item>
          <item row="24" column="1">
           <property name="text">
            <string>eh_nz</string>
           </property>
          </item>
          <item row="25" column="1">
           <property name="text">
            <string>eh_ox</string>
           </property>
          </item>
          <item row="26" column="1">
           <property name="text">
            <string>eh_oy</string>
           </property>
          </item>
          <item row="27" column="1">
           <property name="text">
            <string>eh_oz</string>
           </property>
          </item>
          <item row="28" column="1">
           <property name="text">
            <string>eh_ax</string>
           </property>
          </item>
          <item row="29" column="1">
           <property name="text">
            <string>eh_ay</string>
           </property>
          </item>
          <item row="30" column="1">
           <property name="text">
            <string>eh_az</string>
           </property>
          </item>
          <item row="31" column="1">
           <property name="text">
            <string>eh_x</string>
           </property>
          </item>
          <item row="32" column="1">
           <property name="text">
            <string>eh_y</string>
           </property>
          </item>
          <item row="33" column="1">
           <property name="text">
            <string>eh_z</string>
           </property>
          </item>
          <item row="34" column="0">
           <property name="text">
            <string>标定板[17]位姿</string>
           </property>
          </item>
          <item row="34" column="1">
           <property name="text">
            <string>w_x</string>
           </property>
          </item>
          <item row="35" column="1">
           <property name="text">
            <string>w_y</string>
           </property>
          </item>
          <item row="36" column="1">
           <property name="text">
            <string>w_z</string>
           </property>
          </item>
          <item row="37" column="0">
           <property name="text">
            <string>采图处与标定处距离</string>
           </property>
          </item>
          <item row="37" column="1">
           <property name="text">
            <string>c_dis</string>
           </property>
          </item>
          <item row="38" column="0">
           <property name="text">
            <string>喷涂间隔</string>
           </property>
          </item>
          <item row="38" column="1">
           <property name="text">
            <string>DD</string>
           </property>
          </item>
          <item row="38" column="2">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="39" column="0">
           <property name="text">
            <string>偏移距离</string>
           </property>
          </item>
          <item row="39" column="1">
           <property name="text">
            <string>M_p</string>
           </property>
          </item>
          <item row="39" column="2">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="40" column="0">
           <property name="text">
            <string>机器人准备点</string>
           </property>
          </item>
          <item row="40" column="1">
           <property name="text">
            <string>P000.X</string>
           </property>
          </item>
          <item row="41" column="1">
           <property name="text">
            <string>P000.Y</string>
           </property>
          </item>
          <item row="42" column="1">
           <property name="text">
            <string>P000.Z</string>
           </property>
          </item>
          <item row="43" column="0">
           <property name="text">
            <string>辅相机内参</string>
           </property>
          </item>
          <item row="43" column="1">
           <property name="text">
            <string>Fdx</string>
           </property>
          </item>
          <item row="44" column="1">
           <property name="text">
            <string>u0</string>
           </property>
          </item>
          <item row="45" column="1">
           <property name="text">
            <string>Fdy</string>
           </property>
          </item>
          <item row="46" column="1">
           <property name="text">
            <string>v0</string>
           </property>
          </item>
          <item row="47" column="0">
           <property name="text">
            <string>辅畸变系数</string>
           </property>
          </item>
          <item row="47" column="1">
           <property name="text">
            <string>k1</string>
           </property>
          </item>
          <item row="48" column="1">
           <property name="text">
            <string>k2</string>
           </property>
          </item>
          <item row="49" column="1">
           <property name="text">
            <string>p1</string>
           </property>
          </item>
          <item row="50" column="1">
           <property name="text">
            <string>p2</string>
           </property>
          </item>
          <item row="51" column="0">
           <property name="text">
            <string>光平面方程</string>
           </property>
          </item>
          <item row="51" column="1">
           <property name="text">
            <string>a</string>
           </property>
          </item>
          <item row="52" column="1">
           <property name="text">
            <string>b</string>
           </property>
          </item>
          <item row="53" column="1">
           <property name="text">
            <string>c</string>
           </property>
          </item>
          <item row="54" column="0">
           <property name="text">
            <string>方向向量</string>
           </property>
          </item>
          <item row="54" column="1">
           <property name="text">
            <string>dir_x</string>
           </property>
          </item>
          <item row="55" column="1">
           <property name="text">
            <string>dir_y</string>
           </property>
          </item>
          <item row="56" column="1">
           <property name="text">
            <string>dir_z</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="pixel_x">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="pixel_y">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>图像大小：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="imageSizeX">
         <property name="text">
          <string>800</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>×</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="imageSizeY">
         <property name="text">
          <string>600</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="myLable" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>480</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="pushButton_getStart">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>确认起点</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_getEnd">
         <property name="text">
          <string>确认终点</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_nextLaser">
         <property name="text">
          <string>下一张</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_calibration">
         <property name="text">
          <string>一键标定</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="message">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>myLable</class>
   <extends>QLabel</extends>
   <header>myLable.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
